examples/FIDfromEcho_fail.py

```diff
@@ -0,0 +1,31 @@
+# In[1]:
```

Comment:
Can this be integrated with an existing example? I thought there was one where you were looking over different files?

Reply:
moved into FIDtoEcho_Actual_Challenging.py